docs/tpm.md

```diff
@@ -3,9 +3,9 @@
 
 # Linux* TPM* Implementation
 
-`Ubuntu* OS version 20.04 / RHEL* OS version 8.4` on x86 was used as a development and execution OS. Follow these steps to compile and execute FIDO Device Onboard (FDO).
+`Ubuntu* OS version 20.04 or above / RHEL* OS version 8.4 or above / Debian 11` on x86 was used as a development and execution OS. Follow these steps to compile and execute FIDO Device Onboard (FDO).
```

Comment:
let us call out specific versions

Reply:
@tajnisha , shall we call out 22.04, Debian 11.* instead keeping as 20.04 or above , RHEL can be 8.4 and it is good.